core/tracing/jaeger-osgi/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>org.opennms.core.tracing</artifactId>
+    <groupId>org.opennms.core</groupId>
+    <version>24.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.core.tracing</groupId>
+  <artifactId>org.opennms.core.tracing.jaeger-osgi</artifactId>
+  <packaging>bundle</packaging>
+  <name>OpenNMS :: Core :: Tracing :: Jaeger :: OSGi wrapper</name>
+  <description>OSGi wrapper for Jaeger Tracer.</description>
+
+  <properties>
+    <apacheThriftVersion>0.12.0</apacheThriftVersion>
+  </properties>
+  <build>
+    <defaultGoal>install</defaultGoal>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>2.4.2</version>
```

Comment:
Can we take this opportunity to move the version of the shade plugin the root pom remove the version specifier from everywhere is it currently being used? (Looks like were using a mix of 2.4.2 and 2.4.3 right now)